test/account_multisig.test.js

```diff
@@ -0,0 +1,25 @@
+
+const testUtils  = require('./test-utils');
+const semver = require('semver');
+
+let nearjs;
+let workingAccount;
+let startFromVersion;
+
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 50000;
+
+beforeAll(async () => {
+    nearjs = await testUtils.setUpTestConnection();
+    workingAccount = await testUtils.createAccountMultisig(nearjs, {
+        getCode: () => 'test'
+    });
+    let nodeStatus = await nearjs.connection.provider.status();
+    startFromVersion = (version) => semver.gte(nodeStatus.version.version, version);
+    console.log(startFromVersion);
+});
+
+test('view pre-defined account works and returns correct name', async () => {
```

Comment:
@mattlockyer do you plan to add more tests?

Reply:
Yes working on this right now since it's more critical to release:
https://github.com/near/near-api-js/pull/444#pullrequestreview-522312440

Will be adding more tests tonight.